samples/hssi/hssi_10g_cmd.h

```diff
@@ -352,20 +356,20 @@ class hssi_10g_cmd : public hssi_cmd
     os << "0x3c0d " << std::setw(22) << "pkt_length" << ": " <<
       int_to_hex(hafu->mbox_read(CSR_PACKET_LENGTH)) << std::endl;
     os << "0x3cf4 " << std::setw(22) << "tx_end_tstamp" << ": " <<
-      int_to_hex(hafu->mbox_read(CSR_TX_END_TSTAMP)) << std::endl;
+      int_to_hex(hafu->mbox_read(src_port_, CSR_TX_END_TSTAMP)) << std::endl;
   
     os << "0x3d00 " << std::setw(22) << "num_pkt" << ": " <<
-      int_to_hex(hafu->mbox_read(CSR_NUM_PKT)) << std::endl;
```

Comment:
0x3d00 to 0x3dff all csr access are for dst_port

Reply:
Got it, thanks! 👍